tests/unit_tests/test_array_labels.py

```diff
@@ -36,30 +36,24 @@ def test_array_labels(array_labels_process_code, example_input, expected_output)
     "example_input,raises_exception,error_message",
     [
         ({"data": [1, 0, 3, 2], "labels": [1, 2, 3, 4]}, False, None),
-        ({"data_fake": [1, 0, 3, 2]}, True, "Mandatory argument `data` is not defined."),
+        ({}, True, "Mandatory argument `data` is not defined."),
         ({"data": "[1,0,3,2]"}, True, "Argument `data` is not an array."),
         ({"data": [1, 0, 3, 2]}, True, "Argument `data` is not a labeled array."),
         ({"data": [1, 0, 3, 2], "labels": "[1,2,3,4]"}, True, "Labels in argument `data` is not an array."),
     ],
 )
 def test_array_labels_exceptions(array_labels_process_code, example_input, raises_exception, error_message):
     additional_js_code_to_run = (
-        f"const d = {json.dumps(example_input['data'] if 'data' in example_input else 'undefined')};"
+        f"const d = {json.dumps(example_input['data']) if 'data' in example_input else 'undefined'};"
         + f"d.labels = {json.dumps(example_input['labels']) if 'labels' in example_input else 'undefined'};"
+        if "data" in example_input
+        else ""
+    )
+    process_arguments = f"{{...{json.dumps(example_input)}, 'data': {'d' if 'data' in example_input else 'undefined'}}}"
```

Comment:
Same issue as above. Maybe it would be safer to do this 2 lines up (line 50) in the else statement and just write there `const d = undefined;` (as I am not sure if both `'d'` and `'undefined'` will be processed as strings or variables) and then in this line just pass `d` like `{{... , 'data': d}}`